pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -153,6 +153,10 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 							"/data/db/mongodb-healthcheck",
 							"k8s", "liveness",
 							"--component", "mongos",
+							"--ssl",
+							"--sslCAFile", "/etc/mongodb-ssl/ca.crt",
+							"--sslPEMKeyFile", "/tmp/tls.pem",
+							"--sslInsecure",
```

Comment:
it should be added only if Operator >= 1.7.0

Reply:
@delgod I think you mean >= 1.6.0

Reply:
798d54a